src/context/WalletProvider/KeepKey/utils.ts

```diff
@@ -0,0 +1,6 @@
+import { HDWallet } from '@shapeshiftoss/hdwallet-core'
+import { KeepKeyHDWallet } from '@shapeshiftoss/hdwallet-keepkey'
+
+export const isKeepKeyWallet = (wallet: HDWallet | null): wallet is KeepKeyHDWallet => {
+  return (wallet as KeepKeyHDWallet)?._isKeepKey
```

Comment:
Can't we use `isKeepKey()` instead? That's effectively doing the exact same type guard logic we're doing in `hdwallet`: https://github.com/shapeshift/hdwallet/blob/70bef5b63bd2feaac308ead8e4aa202ba79cfbe4/packages/hdwallet-keepkey/src/keepkey.ts

Reply:
Right you are: https://github.com/shapeshift/web/pull/1341/commits/b35046c7403c58ee9436cda3e8c8d9be2a688ba7